cmd/execution_builder.go

```diff
@@ -711,6 +711,9 @@ func (exeNode *ExecutionNode) LoadIngestionEngine(
 		func() flow.Entity { return &flow.Collection{} },
 		// we are manually triggering batches in execution, but lets still send off a batch once a minute, as a safety net for the sake of retries
 		requester.WithBatchInterval(exeNode.exeConf.requestInterval),
+		requester.WithRetryFunction(requester.RetryLinear(exeNode.exeConf.requestRetryDelay)),
+		requester.WithRetryInitial(exeNode.exeConf.requestRetryInitial),
+		requester.WithRetryMaximum(exeNode.exeConf.requestRetryMaximum),
```

Comment:
When i read requestRetryMaximum, i think of maximum number of retries rather than maximum delay

maybe rename the config fields to requestRetryAdditionalDelay, requestRetryInitialDelay and requestRetryMaximumDelay.